pkg/render/apiserver.go

```diff
@@ -47,30 +47,33 @@ func APIServer(k8sServiceEndpoint k8sapi.ServiceEndpoint, installation *operator
 	tlsHashAnnotations := make(map[string]string)
 	svcDNSNames := dns.GetServiceDNSNames(apiServiceName, APIServerNamespace, clusterDomain)
 
-	if tlsKeyPair == nil {
-		var err error
-		tlsKeyPair, err = CreateOperatorTLSSecret(nil,
-			APIServerTLSSecretName,
-			APIServerSecretKeyName,
-			APIServerSecretCertName,
-			DefaultCertificateDuration,
-			nil,
-			svcDNSNames...,
-		)
-		if err != nil {
-			return nil, err
+	if installation.CertificateManagement == nil {
```

Comment:
```suggestion
	if installation.CertificateManagement == nil && tlsKeyPair == nil {
```

Reply:
I think that will change the result in ways we do not want.
There are three possibilities:
- cert management enabled  => do nothing
- cert management disabled and no tls key pair exists => create key pair and then copy key pair
- cert management disabled and key pair exists => copy key pair